composer.json

```
@@ -31,21 +31,19 @@
         "source": "https://github.com/octobercms/october"
     },
     "require": {
-        "php": ">=7.0.8",
+        "php": "^7.2",
         "ext-mbstring": "*",
         "ext-openssl": "*",
-        "october/rain": "~1.0",
-        "october/system": "~1.0",
-        "october/backend": "~1.0",
-        "october/cms": "~1.0",
-        "laravel/framework": "^5.5.40",
+        "october/rain": "dev-wip/laravel-6",
```

```suggestion
        "october/rain": "dev-wip/laravel-6 as 1.0",
```
The backend modules require 1.0 so I think this is required @bennothommo (correct me if I'm wrong)

---

@LukeTowers you're correct - however, I removed the modules from composer.json temporarily so I didn't have to deal with sub-splitting the branch. My plan was to re-add it back before merging it all in.

---

Ah, I see, makes sense